src/qt/peertablemodel.h

```diff
@@ -88,6 +89,9 @@ public Q_SLOTS:
         /*: Title of Peers Table column which indicates the direction
             the peer connection was initiated from. */
         tr("Direction"),
+        /*: Title of Peers Table column which indicates the duration (length of time)
+            since the peer connection started. */
+        tr("Duration"),
```

Comment:
I like Age, but if you don't, maybe consider Uptime.

Reply:
used Age as column name